cdstub/conf/common

```diff
@@ -0,0 +1 @@
+/home/jeff/mbsssss/common
```

Comment:
All the symbolic links that point to `/home/jeff` will not work on any computer that doesn't have the repository in there. You'll need to recreate these as relative paths with `ln -s ../../common .`